cocos/3d/lights/light-component.ts

```diff
@@ -233,6 +237,21 @@ export class Light extends Component {
         }
     }
 
+    /**
+     * @en Visibility mask of the light, declaring a set of node layers that will be visible to this light.
+     * @zh 光照的可见性掩码，声明在当前光照中可见的节点层级集合。
```

Comment:
Does directional light be effected? If not, should mention it too.

Reply:
sure